packages/labs/nextjs/README.md

```diff
@@ -0,0 +1,37 @@
+# @lit-labs/ssr-react
+
+A package for integrating Lit SSR with Next.js.
+
+## Overview
+
+Lit components, by default, can be imported and added to Next.js projects but will only be shallowly rendered on the server.
+
+This package provides a plugin for Next.js that incorporates tools from [`@lit-labs/ssr-react`](../ssr-react/README.md) into the project for deep server rendering of Lit components.
+
+## Usage
+
+```js
+// next.config.js
+const withLitSSR = require('@lit-labs/nextjs')();
+
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  // Add your own config here
+  reactStrictMode: true,
+  swcMinify: true,
```

Comment:
Are these required?

Reply:
Not required, but they are the default configs that are present when creating a new Next.js project with `create-next-app`.